main/test/ca/uwaterloo/flix/language/phase/TestNamer.scala

```diff
@@ -921,29 +924,30 @@ class TestNamer extends FunSuite with TestUtils {
     expectError[NameError.DuplicateUpperName](result)
   }
 
-  test("DuplicateUpperName.23") {
```

Comment:
Add TODO NS-REFACTOR 

like basically before each test so we dont miss anything.

Reply:
Actually also just make all tests @ignored